common/config/rush/nonbrowser-approved-packages.json

```
@@ -1,7 +1,10 @@
 // DO NOT ADD COMMENTS IN THIS FILE.  They will be lost when the Rush tool resaves it.
 {
   "$schema": "https://developer.microsoft.com/json-schemas/rush/v5/approved-packages.schema.json",
-  "packages": [
+  "packages": [    {
+      "name": "heft-webpack5-plugin",
+      "allowedCategories": [   ]
+    },
```

Should this be here at all? `@rushstack/heft-webpack5-plugin` is included below, not sure why this is here. Also, if it should be included, it's out of alphabetical order. And has no `allowedCategories`.
```suggestion
  "packages": [
```

---

Fixed